web/src/components/profile/ProfileBusinessStructure.test.tsx

```diff
@@ -0,0 +1,71 @@
+import { ProfileBusinessStructure } from "@/components/profile/ProfileBusinessStructure";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ProfileDataContext } from "@/contexts/profileDataContext";
+import { getFlow } from "@/lib/utils/helpers";
+import { useMockRoadmap, useMockRoadmapTask } from "@/test/mock/mockUseRoadmap";
+import { generateProfileData, LookupLegalStructureById } from "@businessnjgovnavigator/shared";
+import { ProfileData } from "@businessnjgovnavigator/shared/profileData";
+import { render, screen } from "@testing-library/react";
+
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+const Config = getMergedConfig();
+
+const renderComponent = (profileData: Partial<ProfileData>): void => {
+  const data = generateProfileData({
+    businessPersona: "STARTING",
+    ...profileData,
+  });
+  render(
+    <ProfileDataContext.Provider
```

Comment:
You should not need to do this yourself - this is what the `WithStatefulProfileData` helper does already